.azure/reusable-build.yml

```
@@ -93,8 +93,21 @@ jobs:
         workingDirectory: $(Build.SourcesDirectory)
         displayName: 'Dump Environment variables'
 
+      - task: NuGetAuthenticate@1
+
+      - script: |
+          copy scripts\onebranch\nuget.config .
+        workingDirectory: $(Build.SourcesDirectory)
+        displayName: 'Copy NuGet.config'
+
+      - script: |
+          nuget install llvm.tools -OutputDirectory packages -version 19.1.4-34 -ExcludeVersion
+          nuget install clang.headers -OutputDirectory packages -version 19.1.4-34 -ExcludeVersion
+        workingDirectory: $(Build.SourcesDirectory)
+        displayName: 'Install LLVM tools'
```

alt: Can avoid this copy with nuget `-ConfigFile` option:

```suggestion
      - script: |
          nuget install llvm.tools -ConfigFile scripts\onebranch\nuget.config -OutputDirectory packages -version 19.1.4-34 -ExcludeVersion
          nuget install clang.headers -ConfigFile scripts\onebranch\nuget.config -OutputDirectory packages -version 19.1.4-34 -ExcludeVersion
        workingDirectory: $(Build.SourcesDirectory)
        displayName: 'Install LLVM tools'
```

---

This change also makes sure that other nuget restores do the right thing and pull from the internal nuget feed.